CONTRIBUTING.md

```diff
@@ -165,4 +166,23 @@ Please keep in mind that the team may **suspend or reject** pull requests. There
 -   Poor quality code.
 -   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.
 
-Please also note that we do not merge pull requests until the changes are documented.
+We merge pull requests after the changes are documented. If you want to document a new capability, add a comment with the description to the pull request. Review our [writing guidelines](#guidelines) before you proceed. 
+
+## Contribute to Documentation
+
+Users can't submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in a private repository. To notify the team of an inaccuracy in the docs, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.
+
+#### Guidelines
+
+TestCafe team adheres to a set of writing guidelines that make our docs easy to read. Follow these rules when you submit written content:
+
+-   Avoid gerunds, passive voice, and past tense.
+-   Avoid double negatives.
+-   Avoid ambiguity, weasel words, and jargon.
```

Comment:
I think we should update "weasel words" because not all readers may be familiar with this term.

```suggestion
-   Avoid misleading or ambiguous words and jargon.
```